web/auth/passphrase.go

```diff
@@ -30,23 +30,27 @@ func passphraseResetForm(c echo.Context) error {
 	if resp, err := couchdb.NormalDocs(instance, consts.BitwardenCiphers, 0, 1, "", false); err == nil {
 		hasCiphers = resp.Total > 0
 	}
-	showBackButton := true
-	if c.QueryParam("hideBackButton") == "true" {
-		showBackButton = false
+	backButton := ""
+	from := c.QueryParam("from")
+	if from != "" {
+		backButton = instance.SubDomain(from).String()
```

Comment:
I'm wondering why we don't build the URL here rather than passing the app's subdomain.